### ISSUE TITLE
Increase no_proxy_match debugging

### ISSUE THREAD

**Issue by lasote** [conan-io/conan#4029]
Print some messages when an URL is being excluded from the proxy. `conan_requester.py` file at line 45.

**Comment by michaelmaguire** [comment]
@michaelmaguire

**Comment by lasote** [comment]
Related with https://github.com/conan-io/conan/issues/4030
This can be done with DEBUG traces.

**Comment by michaelmaguire** [comment]
Is there a way for packaged users of Conan to enable DEBUG traces?  

(In my organization packaging and deployment of Conan is handled by our SREs, and checking out and modification of Conan source is out of scope)

**Comment by lasote** [comment]
yes. setting the environment variable `CONAN_LOGGING_LEVEL=10` will print debug traces to stdout, unset it or set it to 50 will be the default

**Comment by memsharded** [comment]
Closing as old.
